### PR TITLE
Fix section

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: components/content/section.jsx [hunk 1]
```diff
@@ -26,15 +26,7 @@ class Section extends Component {
     if (React.isValidElement(component.props.children))
       return Section.isSectionLike(component.props.children)
 
-    let child
-    try {
-      child = component.type(component.props)
```
Comment (Joozty) [inline components/content/section.jsx:31]: @viktor-yakubiv I am afraid it cannot be used like this because if the component contains hooks it throws `Invalid hooks call error` (see first commit)

I guess we will need a big refactor if we want to use functional components. :(

File: components/content/content.scss [hunk 2]
```diff
@@ -110,7 +111,35 @@ $article-spacer: $section-lg-spacer !default;
...
+        }
+      };
+    }
+  }
+}
+
+.container {
+  padding-left: 0;
+  padding-right: 0;
+}
```
Comment (viktor-yakubiv) [inline components/content/content.scss:139]: This should break the layout on the extra-small screens (mobile, `xs`) where the container is fluid.

File: pages/about/ambassadors.jsx [hunk 3]
```diff
@@ -78,7 +78,7 @@ const AmbassadorsPage = () => (
       </ImageMap>
     </header>
 
-    <Section id="ambassadors-description">
+    <Section id="ambassadors-description" container>
```
Comment (viktor-yakubiv) [inline pages/about/ambassadors.jsx:81]: Do you make container `false` by default?

File: components/content/article.jsx [hunk 4]
```diff
@@ -14,84 +17,64 @@ const ArticleNav = ({ items }) => (
...
+    const headerInner = []
+    const contentInner = []
     React.Children.forEach(children, child => {
       if (!child) return
       if (typeof child.type == 'string' && child.type.match(/header|h[1-6]/))
-        header.push(child)
-      else content.push(React.cloneElement(child, { caption: undefined }))
+        headerInner.push(child)
+      else contentInner.push(child)
     })
```
Comment (viktor-yakubiv) [inline components/content/article.jsx:33]: I think you can avoid this complex logic if you cheat the visual view with column-directed flexbox and order. It would be about 3 rules.

I might have gone in this direction because of accessibility: I thought that the title should go first and the navigation afterwards. However, I am not sure in this, it may not be worth it.

**Review by viktor-yakubiv — COMMENTED** [review]
I would suggest handling containers globally and apply it via CSS only without nested `.container` element. It should simplify the migration. However, what you do is fine too.

File: pages/faq.jsx [hunk 5]
```diff
@@ -21,7 +21,7 @@ const FAQsSection = ({
   return (
     <Section id={id} caption={caption}>
       <Heading>{title}</Heading>
-      <Content>
+      <Content tag="div">
```
Comment (viktor-yakubiv) [inline pages/faq.jsx:24]: Why did you change element type here?

**Review by viktor-yakubiv — APPROVED** [review]
I checked visual look. It's good 👍

File: pages/faq.jsx (hunk 5 above)
Comment (Joozty) [reply pages/faq.jsx]: that was just a leftover of me trying something

**Review by viktor-yakubiv — APPROVED** [review]
I see you are tired of this. The same am I. Perhaps, we can't do better. Let's move this forward! 👍
